refinery/data_set_manager/views.py

```diff
@@ -446,6 +449,20 @@ def post(self, request, *args, **kwargs):
             column.replace('string:', '') for column in source_column_index
         ]
 
+        if settings.REFINERY_DEPLOYMENT_PLATFORM == 'aws':
+            try:
+                identity_id = request.POST.get('identity_id')
+            except (KeyError, ValueError):
+                error = {'error_message': 'identity_id is missing'}
```

Comment:
For some reason, in other places just above it's actually `error_message` for non-AJAX response and `error` for AJAX response which is verbose and confusing, so I thought I'd standardize on `error_message`. I can change it to `error` instead.